src/components/configuration/ConfigBar.tsx

```diff
@@ -77,6 +77,7 @@ function ConfigBar({
                     weekday: zeroIndexedWeekday(weekday),
                     studio: c.location.id,
                     time: { hour, minute },
+                    booking_opens_at: c.bookingOpensAt.toISO(),
```

Comment:
I think `booking_opens_at` should be left out here, and that the booking scheduling should be performed on the backend. In the future, we might even move all this class config info to the backend and only submit a list of ids.